testmain.go

```diff
@@ -51,13 +51,17 @@ type TestingM interface {
 // for any goroutine leaks and fail the tests if any leaks were found.
 func VerifyTestMain(m TestingM, options ...Option) {
 	exitCode := m.Run()
+	opts := buildOpts(options...)
 
 	if exitCode == 0 {
-		if err := Find(options...); err != nil {
+		if err := Find(opts); err != nil {
```

Comment:
Won't this end up passing a `opts` to Find that has teardown set to non-nil? Same for VerifyNone.